src/views/TabView/TabView.js

```diff
@@ -56,7 +54,14 @@ type Props = {
   },
 };
 
-class TabView extends PureComponent<void, Props, void> {
+class TabView extends PureComponent<$Shape<Props>, Props, void> {
+  static defaultProps = {
+    // fix for https://github.com/react-native-community/react-native-tab-view/issues/312
+    initialLayout: Platform.select({
+      android: { width: 1, height: 0 },
```

Comment:
Please be explicit and include `ios: undefined`. Thanks

Reply:
I should add `web: undefined` also then?

Reply:
I see your point. Let's keep it.